app/src/main/java/com/fivegmag/a5gmsmediasessionhandler/MediaSessionHandlerMessengerService.kt

```diff
@@ -37,9 +31,10 @@ import retrofit2.Call
 import retrofit2.Response
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
-
-import java.util.Timer
-import java.util.TimerTask
+import java.lang.Long.min
+import java.text.SimpleDateFormat
+import java.util.*
```

Comment:
Please only import the classes that you need instead of the whole package

Reply:
done